packages/teams-js/test/private/files.spec.ts

```diff
@@ -1,14 +1,22 @@
-import { files } from '../../src/private/files';
```

Comment:
Please make sure this file matches what's checked in to 2.0-preview. Lakhveer recently made changes and we don't want to undo them.